site/src/pages/docs/advanced-unit-testing.mdx

```diff
@@ -9,28 +9,28 @@ A unit test of a client or a service will require you to prepare two objects:
 object easily:
 
 ```java
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 import com.linecorp.armeria.common.HttpRequest;
 import com.linecorp.armeria.common.HttpResponse;
 import com.linecorp.armeria.common.AggregatedHttpResponse;
 import com.linecorp.armeria.client.ClientRequestContext;
 import com.linecorp.armeria.server.ServiceRequestContext;
 
-public class MyJUnit4Test {
+class MyJUnit5Test {
 
     private MyClient client;
     private MyService service;
 
-    @Before
+    @BeforeEach
     public void setUp() {
         client = ...;
         service = ...;
     }
 
     @Test
-    public void testClient() throws Exception {
+    void testClient() throws Exception {
```

Comment:
Could remove?
```suggestion
    void testClient() {
```

Reply:
Changed `.get()` to `.join()` and removed it. Thanks!